doc/reference/power_management/index.rst

```diff
@@ -395,6 +395,44 @@ Check Busy Status of All Devices API
 
 Checks if any device is busy. The API returns 0 if no device in the system is busy.
 
+Wakeup capability
+-----------------
+
+Some devices are capable of waking the system up from a sleep state.
+When a device has such capability, applications can enable or disable
+this feature on a device dynamically using
+:c:func:`pm_device_wakeup_enable`.
+
+This property can be set on device declaring the property ``wakeup-source`` in
+the device node in devicetree. For example, this devicetree fragment sets the
+``gpio0`` device as a "wakeup" source:
+
+.. code-block:: devicetree
+
+		gpio0: gpio@40022000 {
+			compatible = "ti,cc13xx-cc26xx-gpio";
+			reg = <0x40022000 0x400>;
+			interrupts = <0 0>;
+			status = "disabled";
+			label = "GPIO_0";
+			gpio-controller;
+			wakeup-source;
+			#gpio-cells = <2>;
+		};
+
+By default, "wakeup" capable devices are not enabled
+during the device initialization. Applications can enable this
+functionality later calling :c:func:`pm_device_wakeup_enable`.
```

Comment:
This is confusing to me as an outside reader. It kind of sounds like you're saying the device will not be initialized. I don't know what "enabling" a device "during the device initialization" means except for initializing it. What am I missing here? Is there some way to make it clearer?

Not a blocking comment, but it would be nice to get some clarification here.

Reply:
that makes sense. I've read it again and it is indeed confuse.

Will change to;
```
... capable devices do not have this functionality enabled during the device initialization....
```